protocol/protocolwrapper/protocol_filter_wrapper_test.go

```diff
@@ -62,12 +62,12 @@ func TestProtocolFilterWrapperRefer(t *testing.T) {
 
 // the same as echo filter, for test
 func init() {
-	extension.SetFilter("echo", GetFilter)
+	extension.SetFilter("echo", newFilter)
 }
```

Comment:
Is this not affected the real EchoFilter? (If this `SetFilter` is performed after real EchoFilter 's `SetFilter`) @LaurenceLiZhixin